src/cssUsage.js

```diff
@@ -605,69 +605,82 @@ void function() { try {
 		/**
 		 * This will transform a value into an array of value identifiers
 		 */ 
-		function createValueArray(value, propertyName, dontNormalize = true) {
+		function createValueArray(value, propertyName, normalize = true) {

```

Comment:
Can we instead keep the name "dontNormalize" and change the default value to false, then look in the code when we specify the argument and invert its value there? Default arguments with a default value of true is kinda wonky.

Reply:
Yeah we can do that.